holoviews/plotting/widgets/__init__.py

```diff
@@ -194,7 +194,9 @@ def _plot_figure(self, idx):
 
 
     def update(self, key):
-        return self._plot_figure(key)
+        self.plot.update(key)
+        self.plot.push()
+        return 'Complete'
```

Comment:
Currently part of the protocol, which will change as soon as we use comms for everything.